codespell_lib/data/dictionary.txt

```diff
@@ -32119,9 +32119,11 @@ wayoint->waypoint
 wayoints->waypoints
 wayword->wayward
 weant->want, wean,
+weaponaries->weaponries
```

Comment:
I like @vikivivi solution as long as weaponries is a valid correction.

Reply:
I've chanegd this to `weaponaries->weaponry, weaponries,`